### PR TITLE
Resend verification code

### PR DESCRIPTION
**Pull request by fremartini**

### CODE REVIEW

**Comment by fremartini** [comment]
@marfavi two elements need styling. Can you do it?

File: lib/features/login/presentation/pages/login_page_passcode.dart [hunk 1]
```diff
@@ -34,14 +33,35 @@ class _LoginPagePasscodeState extends State<LoginPagePasscode> {
...
+    await appDialog(
+      context: context,
+      title: Strings.loginVerificationEmailSent,
+      children: [
+        //FIXME: style
+        Text(Strings.loginVerificationEMailBody(widget.email)),
+      ],
+      actions: [
+        TextButton(
+          //FIXME: style
```
Comment (fremartini) [inline lib/features/login/presentation/pages/login_page_passcode.dart:47]: .

File: lib/features/login/presentation/pages/login_page_passcode.dart [hunk 2]
```diff
@@ -34,14 +33,35 @@ class _LoginPagePasscodeState extends State<LoginPagePasscode> {
     );
   }
 
+  Future<void> resendEmailCallback(LoginCubit cubit) async {
+    cubit.resendVerificationEmail(widget.email);
+    await appDialog(
+      context: context,
+      title: Strings.loginVerificationEmailSent,
+      children: [
+        //FIXME: style
```
Comment (fremartini) [inline lib/features/login/presentation/pages/login_page_passcode.dart:42]: .

File: lib/base/strings.dart [hunk 3]
```diff
@@ -30,6 +30,11 @@ abstract final class Strings {
   static const loginForgotYourPasscode = 'Forgot your passcode?';
   static const loginForgot = 'Forgot?';
 
+  static const loginResendVerificationEmail = 'Resend verification email';
+  static const loginVerificationEmailSent = 'Verification email sent';
+  static String loginVerificationEMailBody(String email) =>
```
Comment (fredpetersen) [inline lib/base/strings.dart:35]:
```suggestion
  static String loginVerificationEmailBody(String email) =>
```
Of course this also applies anywhere it's used

**Review by fredpetersen — COMMENTED** [review]
When dialog is styled, and variable name is addressed, this seems good to me :)

**Comment by fremartini** [comment]
@fredpetersen styling should be part of another PR

**Review by fredpetersen — APPROVED** [review]
Looks good now :)

Did you make an issue for styling this later?

**Pull request merged**